phoenix-core/src/test/java/org/apache/phoenix/compile/RVCOffsetCompilerTest.java

```diff
@@ -148,12 +153,82 @@ public void buildListOfRowKeyColumnExpressionsIndexTest() throws Exception {
         expressions.add(expression1);
         expressions.add(expression2);
 
+        AndExpression expression = mock(AndExpression.class);
+        Mockito.when(expression.getChildren()).thenReturn(expressions);
+
+        List<RowKeyColumnExpression>
+                result =
+                offsetCompiler.buildListOfRowKeyColumnExpressions(expression, true);
+
+        assertEquals(2,result.size());
+        assertEquals(rvc1,result.get(0));
+        assertEquals(rvc2,result.get(1));
+    }
+
+    @Test
+    public void buildListOfRowKeyColumnExpressionsSingleNodeComparisonTest() throws Exception {
+        List<Expression> expressions = new ArrayList<>();
+
+        RowKeyColumnExpression rvc1 = new RowKeyColumnExpression();
+        RowKeyColumnExpression rvc2 = new RowKeyColumnExpression();
+
+        ComparisonExpression expression1 = mock(ComparisonExpression.class);
+
+        Mockito.when(expression1.getChildren()).thenReturn(Lists.<Expression>newArrayList(rvc1));
+
+        List<RowKeyColumnExpression>
+                result =
+                offsetCompiler.buildListOfRowKeyColumnExpressions(expression1, false);
+
+        assertEquals(1,result.size());
+        assertEquals(rvc1,result.get(0));
+    }
+
+    @Test
+    public void buildListOfRowKeyColumnExpressionsSingleNodeIsNullTest() throws Exception {
+        List<Expression> expressions = new ArrayList<>();
+
+        RowKeyColumnExpression rvc1 = new RowKeyColumnExpression();
+        RowKeyColumnExpression rvc2 = new RowKeyColumnExpression();
+
+        IsNullExpression expression1 = mock(IsNullExpression.class);
```

Comment:
nit: renaming to `expression` since no expression2 here

Reply:
will do